portal2-internal/hack/cfg/opts.hpp

```diff
@@ -16,6 +16,10 @@ namespace opts {
         color_t m_portal_2 = {100, 255, 255, 255};
 
         color_t& operator[](size_t index) {
+            return at(index);
+        }
+
+        color_t& at(size_t index) {
             switch (index) {
             case 0:
```

Comment:
kinda hate how portal 1 is at the index 0..

hb a `enum portal` with literally portal1 and portal2

Reply:
we treat the portal option struct as an array where indexation starts at 0, but yeah sure we can also add an enum with said indices